File: app/src/main/java/com/typ/muslim/core/praytime/enums/AsrMethod.java
package com.typ.muslim.core.praytime.enums;

/**
 * AsrMethod used during calculating PrayTimes
 */
public enum AsrMethod {
    /**
     * Shafii
     */
    SHAFII,
    /**
     * Hanafi
     */
    HANAFI;

    public static AsrMethod valueOf(int ordinal) {
        if (ordinal == 1) return HANAFI;
        else return SHAFII;
    }

}

File: app/src/main/java/com/typ/muslim/features/prays/enums/AsrMethod.kt
package com.typ.muslim.features.prays.enums

/**
 * AsrMethod used during calculating PrayTimes
 */
enum class AsrMethod {
    /**
     * Shafii
     */
    SHAFII,

    /**
     * Hanafi
     */
    HANAFI;

    companion object {
        @JvmStatic
        fun valueOf(ordinal: Int): AsrMethod {
            return if (ordinal == 1) HANAFI else SHAFII
        }
    }
}
